packages/ai-vercel/src/react/interrupts.ts

```diff
@@ -49,26 +53,26 @@ export const useInterruptions = (
         error.message.replace(InterruptionPrefix, "")
       );
       const { id } = parsedError.toolCall;
-
       setToolInterrupt({
         ...parsedError,
         resume: (result: any) => {
           setToolInterrupt(null);
-
           if (parsedError?.behavior === "reload") {
             regenerate();
           } else {
             addToolResult({
-              // @ts-expect-error toolCall
+              // added for compatibility w/ SDK5
+              tool: parsedError.toolCall?.name,
```

Comment:
added for SDK5 shape:
https://ai-sdk.dev/docs/migration-guides/migration-guide-5-0#addtoolresult-changes

We are still referencing `toolName` w/ several existing types. This could be revisited.